WORKSPACE

```diff
@@ -120,11 +120,45 @@ annex_scalafmt_pinned_maven_install()
 
 scalafmt_default_config()
 
+load("//:workspace.bzl", "hocon_repositories")
+
+hocon_repositories()
+
+load("@hocon_maven//:defs.bzl", hocon_maven_install = "pinned_maven_install")
+
+hocon_maven_install()
+
 bind(
     name = "default_scala",
-    actual = "@rules_scala_annex//src/main/scala:zinc_2_12_8",
```

Comment:
If it was default to 2.12, how come it is not compatible with 2.12?

Reply:
This wasn't being used before. There are no tests in the rule set so the hocon compiler wasn't in use here, and we specify Scala 2.11 in the repo that consumes this rule set.